modules/rosa-installing-prerequisites.adoc

```diff
@@ -4,30 +4,31 @@
 // getting_started_rosa/rosa-creating-cluster.adoc
 
 
-[id="rosa-installing-prerequisites"]
+[id="rosa-installing-prerequisites_{context}"]
 = Installing prerequisites
 
 Complete the following prerequisites before creating your {product-title} cluster.
 
 .Procedure
 
-. Select the AWS account you want to use.
+. Create a link:https://cloud.redhat.com[Red Hat account], if you do not already have one. Then, check your email for a verification link. You will need the credentials to install ROSA.
+. Select the Amazon Web Services (AWS) account that you want to use.
 +
-Unless you are just testing out {product-title}, it is recommended to use a dedicated AWS account to run any production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
+Unless you are only testing {product-title}, it is recommended to use a dedicated AWS account to run any production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
 +
-If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
+If you are using AWS Organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
```

Comment:
This link will need to change when we have the other CCS doc in the ROSA docs.

Reply:
This will have to be done in a new JIRA: https://issues.redhat.com/browse/OSDOCS-1802